src/UI/Component/Dropzone/File/Dropzone.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Interface Dropzone
+ *
+ * Describes a dropzone that can be configured to use the darkened background
+ * or not.
```

Comment:
The darkened background uses the same bootstrap classes like the modals. So there is no less variable for it.

The current dropzones that are available in ILIAS use that yellow background style. In the feature wiki [File Upload for all Appointment Types](http://www.ilias.de/docu/goto_docu_wiki_wpage_4397_1357.html) there are dropzones in the calendar that use the darkened background. If the highlighted background is defined with less, it would not be possible to use both. Therefore, some rules to decide when we use the highlighted background or not would be good.

The dropzone interface is in the file level, because other possible dropzones and how they work or what they do are to unknown at the moment. But to even use this interface depends on the decision how we want to configure the darkened background, with less or with the interface.